app/components/Swapbox.tsx

```diff
@@ -362,30 +371,20 @@ export const Swapbox = ({ fixedProductMarketMaker }: SwapboxProps) => {
               </div>
             )}
           </div>
-          {isDisconnected ? (
-            <ConnectButton width="full" size="lg">
-              Connect
-            </ConnectButton>
-          ) : unsupportedNetwork ? (
-            <Button width="full" variant="pastel" size="lg" disabled>
-              Unsupported network
-            </Button>
-          ) : !tokenAmountIn || +tokenAmountIn === 0 ? (
-            <Button width="full" variant="pastel" size="lg" disabled>
-              Enter amount
-            </Button>
-          ) : currentState.isLoading ? (
+          {currentState.isLoading ? (
             <Button width="full" variant="pastel" size="lg" disabled>
               Fetching price
             </Button>
```

Comment:
We isn't this inside the `getButtonLabel` and `disable` logic? I think we can add it there too